gems/pending/Gemfile

```diff
@@ -21,7 +21,7 @@ gem "excon",                   "~>0.40",            :require => false
 gem "ezcrypto",                "=0.7",              :require => false
 gem "ffi",                     "~>1.9.3",           :require => false
 gem "ffi-vix_disk_lib",        "~>1.0.2",           :require => false  # used by lib/VixDiskLib
-gem "fog",                     "~>1.37.0",          :require => false
+gem "fog-openstack",           "~>0.1.2",           :require => false
 gem "fog-core",                "!=1.31.1",          :require => false
```

Comment:
@erjohnso yes, removing it for now, since it didn't point to a specific version anyway